05_ANALYSIS_TOOLS/etl_pipeline/load/json_loader.py

```
@@ -0,0 +1,637 @@
+                self._load_business_data(transformed_data['business_metrics'])
+            
+            # Load coverage analysis
+            if 'coverage_analysis' in transformed_data:
+                self._load_coverage_analysis(transformed_data['coverage_analysis'])
+            
+            # Load equipment data
+            if 'equipment_data' in transformed_data:
+                self._load_equipment_data(transformed_data['equipment_data'])
+            
+            logger.info("JSON data loading completed")
+            return self.load_results
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)